about.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: docs
+title: About
+headline: All About Pico
+description: What *is* a "stupidly simple, blazing fast, flat file CMS" anyway?
+toc:
+  a-flat-file-cms: A Flat File CMS
+  web-pages-without-html:
+    _title: Web Pages Without HTML
+    markdown-makes-formatting-easy: Markdown Makes Formatting Easy
+    control-your-metadata-with-yaml: Control Your Metadata With YAML
+  flexibility-and-customization:
+    _title: Flexibility and Customization
+    themes: Themes...
+    plugins: Plugins...
+  getting-started: Getting Started
+nav-url: /about/
+nav: 2
+---
+
+Pico is a stupidly simple, blazing fast, flat file CMS.  For those of us who aren't an expert webmaster, what does that even mean?  In the most basic sense, it means that there is no administration backend or database to deal with. You simply create .md files in the content folder and those files become your pages.  There's *much* more to Pico than that though.
+
+![Pico's Default Theme]({{ site.github.url }}/style/images/docs/pico_about_default_theme.png)
+Pico's default theme.  It's a bit bare... but that's intentional!
+
+* Center the text below images.
+
+Let's get this out of the way: Pico is *not* a turn-key solution.  If your goal is to deploy a generic-looking website without getting your hands dirty, Pico is probably not for you.
+
+* Is this too discouraging?
+
+That being said, Pico is incredibly extendable and customizable.  With a very small amount of configuration, you'll find yourself with a very personalized experience.  There's also a growing community creating ready-to-go [themes and plugins](#flexibility-and-customization) for Pico.  It is very possible to get started with Pico even with [no prior knowledge of HTML](#web-pages-without-html).  You'll find however that a little bit of coding knowledge will get you a long way.
+
+![NotePaper Theme for Pico]({{ site.github.url }}/style/images/docs/pico_about_notepaper_theme.jpg)
+NotePaper, a ready-to-go, community-developed theme.
+
+* Probably move this later, but I didn't want to forget to include it in the same commit as the image file.
+* Add example images for other themes.
+
+
+* insert some images showing Pico in use.  Borrow from readme?
+  * Can the site do thumbnails and lightboxing?
```

Comment:
Unfortunately not (as far as I know). You'll need to read and adapt the HTML sourcecode manually :unamused: